osu.Game.Rulesets.Catch/Mods/CatchModFlashlight.cs

```diff
@@ -15,9 +16,26 @@ public class CatchModFlashlight : ModFlashlight<CatchHitObject>
     {
         public override double ScoreMultiplier => 1.12;
 
-        private const float default_flashlight_size = 350;
+        [SettingSource("Flashlight size", "Multiplier applied to the default flashlight size.")]
+        public override BindableNumber<float> SizeMultiplier { get; } = new BindableNumber<float>
```

Comment:
Is there a reason these have to be different per ruleset? If they do, can the defaults not be overridden in a ctor rather than having these abstract?

Reply:
yes, they do have to be different, since the base radiuses are different for each mode, unless im misunderstanding you

As for why they are overriden in this manner its simply because thats how ( atleast to  my understanding) i was told to do it by bdach and frenzibyte